.github/workflows/test.yml

```diff
@@ -0,0 +1,19 @@
+name: Test
```

Comment:
It doesn't bother me, so only mentioning in case it does you, but the job name here is `Test`, whereas that for the sibling runtime repo is `Build`, even though the workflows themselves are very similar.

https://github.com/forcedotcom/sf-fx-runtime-nodejs/pull/436/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R1

Reply:
it does bother me but i was using this repository as a light introduction to GitHub Actions so i didn't clue into how this showed up in the UI initially.  i was going to leave it though since this repo will no longer require building after #146 is merged.